Portal/src/Datahub.Portal/Layout/Topbar/Topbar.razor

```diff
@@ -13,7 +13,14 @@
         </li>
     </ul>
 </nav>
-<MudImage Src="/img/canada-fip.svg" Alt=@Localizer["Government of Canada Wordmark"] Style="max-height: 25px; float:left!important;"/>
+@if (Localizer["_currentLang"] == "en"){
+    <MudImage Src="/img/canada-fip.svg" Alt=@Localizer["Government of Canada Wordmark"] Style="max-height: 25px; float:left!important;"/>
```

Comment:
probably could have leveraged your localizer hack and used it for the Src as well :-)

Reply:
Also we probably have a helper method somewhere to test the current locale. I don't mind using this _currentLanguage if we document it clearly

Reply:
> probably could have leveraged your localizer hack and used it for the Src as well :-)

Did this to clean it up, great idea